apps/web/src/features/organizations/hooks/useSiwe.tsx

```diff
@@ -0,0 +1,53 @@
+import { useWeb3 } from '@/hooks/wallets/web3'
+import { useAuthGetNonceV1Query, useAuthVerifyV1Mutation } from '@safe-global/store/gateway/AUTO_GENERATED/auth'
+import { useCallback } from 'react'
+
+export const useSiwe = () => {
```

Comment:
Can we replace `signInWithEthereum` with this hook and move the hook to the general `services/siwe` folder?